sli/tokenstack.h

```diff
@@ -160,7 +160,7 @@ class TokenStack : private TokenArrayObj
   void
   roll( size_t n, long k )
   {
-    if ( n < 2 || k == 0 )
+    if ( n < 2 and k == 0 )
```

Comment:
```suggestion
    if ( n < 2 or k == 0 )
```

Reply:
Awesome, that's it! No idea how that happened, but glad you found it. Checks pass now locally already.